src/main/kotlin/xyz/xenondevs/nova/jetpacks/registry/Abilities.kt

```diff
@@ -8,6 +9,10 @@ import xyz.xenondevs.nova.jetpacks.Jetpacks
 import xyz.xenondevs.nova.jetpacks.ability.JetpackFlyAbility
 import xyz.xenondevs.nova.util.data.getFloat
 
+private val IGNORED_GAME_MODES = configReloadable {
+    val modesRaw = NovaConfig["jetpacks:config"].getStringList("ignored_game_mode")
+    GameMode.values().filter { gameMode -> modesRaw.any { it.equals(gameMode.name, ignoreCase = true) } }
+}
```

Comment:
This should probably be in JetpackFlyAbility if it's the same across all jetpack types